### DISCUSSION TITLE
Tor dietro Firewall

### DISCUSSION THREAD

**Discussion by f0r7y** [globaleaks/globaleaks-whistleblowing-software#3552] [Q&A]
Buongiorno
ho installato l'ultima versione di Globaleaks su una VM Ubuntu.
La connessione dal web classico funziona correttamente, mentre da TOR torna il messaggio

> Sito Onion Non Trovato
> La causa più probabile è che il sito onion sia offline. Contatta l'amministratore del sito onion.
> Dettagli: 0xF0 — Il descrittore del servizio onion richiesto non può essere trovato nell'hashring, perciò il servizio non è raggiungibile dal client.

Nel file globaleaks.log, trovo l'errore
`2023-07-24 09:28:52+0000 [-] [E] Failed to initialize Tor connection; error: Tor control port not open on /var/run/tor/control; waiting for Tor to become available`

Probabilmente qualche porta va aperta, quali sono?
Altrimenti, quale configurazione di rete consigliate?
Grazie

**Comment by evilaliv3** [comment]
Grazie per la domanda.

Come hai creato la VM?

**Comment by f0r7y** [comment]
Ho messo su Ubuntu 22.04.2 LTS e ho seguito la guida per l'installazione presente nella documentazione di Globaleaks.
Ora la VM può comunicare in uscita sulle porte 587, 443, 53 e 123. in ingresso solo sulla 443. Quali altre porte servono?

**Reply by alobefaro** [reply]
Mi ritrovo nella stessa identica situazione. Come ulteriore indicazione, oltre ad aver aggiornato all'ultima versione, ho fatto aprire tutte le porte ma nessun risultato. In alternativa, è possibile eliminare la sezione in home page per l'uso di TOR sino al. ripristino della funzionalità?
